docker/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+services:
+  ros2bridge20.04:
+    build:
+      context: "."
+      dockerfile: "./Dockerfile"
+    environment:
+      DISPLAY: $DISPLAY
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix
+      - ~/.Xauthority:/home/$USER/.Xauthority:rw
```

Comment:
Not necessary since we are not visualizing anything from this image

Reply:
This line was already removed locally. Will push when the commit is ready.